client/methods.js

```diff
@@ -1,9 +1,11 @@
 import { $ } from 'meteor/jquery';
+import { Meteor } from 'meteor/meteor';
```

Comment:
Cool improvement. Need to check if istanbul can still access to these methods with Package...